File: src/main/java/com/Angus/guess/GameFor.java
package com.Angus.guess;

import java.util.Random;
import java.util.Scanner;

public class GameFor {
    public static void main(String[] args) {

        //random secret
        Random random = new Random();
        int secret = random.nextInt(10) + 1;
        System.out.println(secret);

        //scanner number

        for (int i = 1; i <= 4; i++){
            System.out.println("Please enter the numer :");
            Scanner scanner = new Scanner(System.in);
            int number = scanner.nextInt();
            System.out.print("第" + i + "次:" + number);
                if (secret > number)
                    System.out.println("Higher");
                else if (secret < number)
                    System.out.println("Lower");
                else{
                    System.out.println("Great , the secert is :" + secret);
                    break;
                }
        }
        System.out.println("The coding is ending");

    }
}

File: src/main/kotlin/com/Angus/GameFor.kt
package com.Angus

import java.util.*
import kotlin.random.Random

fun main(args: Array<String>) {
    val scanner = Scanner(System.`in`)
    val secret : Int = Random.nextInt(10) + 1
    println(secret)
    for (i in 1..5){
        var number = scanner.nextInt();
        println("Please enter the number (${i}/4): ")
        println("第${i}次: $number")
        if (secret > number)
            println("Higher")
        else if (secret < number)
            println("Lower")
        else{
            println("Great , the number is $number")
            break
        }
    }
    println("The coding is ending")
}